contracts/ERC20Safe.sol

```diff
@@ -36,11 +36,17 @@ contract ERC20Safe is Initializable, BridgeRole, Pausable {
     uint64 public depositsCount;
     uint16 public batchSize;
     uint16 private constant maxBatchSize = 100;
-    uint8 public batchBlockLimit;
-    uint8 public batchSettleLimit;
+    uint8 public oldBatchBlockLimit;  // unused
+    uint8 public oldBatchSettleLimit; // unused
+
+    uint16 public batchBlockLimit;
+    uint16 public batchSettleLimit;
 
     // Reserved storage slots for future upgrades
-    uint256[10] private __gap;
+    uint32 private __gap32;
+    uint64 private __gap64;
+    uint128 private __gap128;
```

Comment:
You can make the uint64 an array with capacity of three (not a strong opinion)

Reply:
let's try :fire: